src/html/transfers.html

```diff
@@ -332,6 +382,23 @@ <h1>Your Transfers</h1>
     footer.classList.toggle('open')
     adjustTransferDetailsHeights()
   })
+  document.querySelector('body').addEventListener('click', event => {
+    const deleteButton = event.target.closest('[data-behavior=deleteTransfer]')
+
+    // no delete button clicked, end here
+    if (!deleteButton) return
+
+    window.dom.show('deleteModal')
+    transferIdToRemove = deleteButton.closest('[data-behavior=transfer]').id
```

Comment:
I think you could add `confirmDeleteButton.onclick` here and remove the quasi-global, since subsequent clicks of a delete button would override previous `onclick` handlers. (Using `addEventListener` wouldn't work.)